qiime2/core/testing/examples.py

```diff
@@ -50,6 +50,26 @@ def mdc1_factory():
                                                     name='id')))
 
 
+def intSequence_list_factory():
```

Comment:
Please reformat this as a [PEP 8-compliant name](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

```suggestion
def int_sequence_list_factory():
```